sources/en/w/wuxiasky.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from sources.en.a.asianovel_net import AsiaNovelNetCrawler
```

Comment:
it is better to not depend on a file inside the sources folder. for auto updates, it will create an issue. you can put this link inside the AsiaNovelNetCrawler itself. i.e.

```
class AsiaNovelNetCrawler(SearchableSoupTemplate, ChapterOnlySoupTemplate):
    base_url = [
        "https://www.asianovel.net/",
        "https://www.wuxiasky.net",
    ]
```

Reply:
or, if this template is a common one, we can put it inside `lncrawl/templates` as well